src/main/resources/logback.xml

```diff
@@ -0,0 +1,12 @@
+<configuration>
+    <appender name="LOG_DATA_MASK" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
Rename this appender name since no masking is perfomed at all here so LOG_DATA_MASK can be confusing

Reply:
Fixed with cf09045e15b786547893b027a48e8eba1fbe1f30